### ISSUE TITLE
Why all the new permissions? 

### ISSUE THREAD

**Issue by markusschloesser** [CDrummond/lms-material-app#21]
I just updated from f-droid and was wondering about all the (new) permissions?
![Screenshot_20240615-010509.png](https://github.com/CDrummond/lms-material-app/assets/59286549/3d4a822a-886f-4ab8-a89e-50c8f0387e58)

**Comment by CDrummond** [comment]
There a are no new permissions between 0.4.2 and 0.5.0.

Storage permissions are for downloading files, and probably also for WebView cache.
WiFi is for detetcting if connected, and for enabling reconnect.
Termux commands are for running Squeezelite in Termux.

**Comment by byles** [comment]
Sorry Mr Craig for commenting on a closed issue, but this might be permissions related and until I do some actual proper checking I didn't want to open another issue... Android 11 on the Moto g10, the "Show notification" setting doesn't prompt for notification permission as it did on Android 13, and there are no notifications. The reason I comment on this issue in particular is that I don't see any permissions listed as above at all on my Android 11. I tried the player by termux too actually, but couldn't get it working. The setting just kept reverting to None. I noticed that related permission  in the screenshot above.
